plantation_mapping/core.py

```diff
@@ -0,0 +1,501 @@
+    PRIMITIVE_CROPLAND = ee.ImageCollection('projects/servir-mekong/yearly_primitives_smoothed/cropland')
+    PRIMITIVE_GRASS = ee.ImageCollection('projects/servir-mekong/yearly_primitives_smoothed/grass')
+    PRIMITIVE_MANGROVE = ee.ImageCollection('projects/servir-mekong/yearly_primitives_smoothed/mangrove')
+    PRIMITIVE_OPEN_FOREST = ee.ImageCollection('projects/servir-mekong/yearly_primitives_smoothed/openForest')
+    PRIMITIVE_SNOW = ee.ImageCollection('projects/servir-mekong/yearly_primitives_smoothed/snow')
+    PRIMITIVE_URBAN = ee.ImageCollection('projects/servir-mekong/yearly_primitives_smoothed/urban')
+    PRIMITIVE_WATER = ee.ImageCollection('projects/servir-mekong/yearly_primitives_smoothed/water')
+    PRIMITIVE_WETLANDS = ee.ImageCollection('projects/servir-mekong/yearly_primitives_smoothed/wetlands')
+    PRIMITIVE_WOODY = ee.ImageCollection('projects/servir-mekong/yearly_primitives_smoothed/woody')
+    PRIMITIVES = [
+        PRIMITIVE_CLOSED_FOREST, PRIMITIVE_CROPLAND, PRIMITIVE_GRASS,
+        PRIMITIVE_MANGROVE, PRIMITIVE_OPEN_FOREST, PRIMITIVE_SNOW, PRIMITIVE_URBAN,
+        PRIMITIVE_WATER, PRIMITIVE_WETLANDS, PRIMITIVE_WOODY
+    ]
+
+    # geometries
+    MEKONG_FEATURE_COLLECTION = ee.FeatureCollection('ft:1tdSwUL7MVpOauSgRzqVTOwdfy17KDbw-1d9omPw')
+    DEFAULT_GEOM = MEKONG_FEATURE_COLLECTION.filter(ee.Filter.inList('Country', ['Myanmar (Burma)'])).geometry()
+
+    # Class and Index
+    LANDCOVERCLASSES = [
+        {
+            'name': 'Unknown',
+            'value': '0',
+            'color': '6f6f6f'
+        },
+        {
+            'name': 'Surface Water',
+            'value': '1',
+            'color': '0000ff'
+        },
+        {
+            'name': 'Snow and Ice',
+            'value': '2',
+            'color': '808080'
+        },
+        {
+            'name': 'Mangroves',
+            'value': '3',
+            'color': '556b2f'
+        },
+        {
+            'name': 'Cropland',
+            'value': '4',
+            'color': '7cfc00'
+        },
+        {
+            'name': 'Urban and Built up',
+            'value': '5',
+            'color': '8b0000'
+        },
+        {
+            'name': 'Grassland',
+            'value': '6',
+            'color': '20b2aa'
+        },
+        {
+            'name': 'Closed Forest',
+            'value': '7',
+            'color': '006400'
+        },
+        {
+            'name': 'Open Forest',
+            'value': '8',
+            'color': '90ee90'
+        },
+        {
+            'name': 'Wetland',
+            'value': '9',
+            'color': '42f4c2'
+        },
+        {
+            'name': 'Woody',
+            'value': '10',
+            'color': '8b4513'
+        },
+        {
+            'name': 'Other land',
+            'value': '11',
+            'color': '6f6f6f'
+        }
+    ]
+
+    REMAPPED_CLASSES = [
```

Comment:
we are not remapping any classes here. look the examples at the myanmar products